src/clouds.cpp

```diff
@@ -113,19 +123,19 @@ void Clouds::render()
 		Clouds move from Z+ towards Z-
 	*/
 
-	static const float cloud_size = BS * 64.0f;
-	
 	const float cloud_full_radius = cloud_size * m_cloud_radius_i;
 	
+	v2f camera_pos_2d(m_camera_pos.X, m_camera_pos.Z);
 	// Position of cloud noise origin from the camera
-	v2f cloud_origin_from_camera_f = m_origin - m_camera_pos;
+	v2f cloud_origin_from_camera_f = m_origin - camera_pos_2d;
 	// The center point of drawing in the noise
 	v2f center_of_drawing_in_noise_f = -cloud_origin_from_camera_f;
 	// The integer center point of drawing in the noise
 	v2s16 center_of_drawing_in_noise_i(
-		MYROUND(center_of_drawing_in_noise_f.X / cloud_size),
-		MYROUND(center_of_drawing_in_noise_f.Y / cloud_size)
+		floor(center_of_drawing_in_noise_f.X / cloud_size),
```

Comment:
`error: ‘floor’ is not a member of ‘std’` – I had included `<math.h>`, it works with `<cmath>`. Hope that's ok?